com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDBlockFields.cs

```diff
@@ -11,6 +11,9 @@ public struct VertexDescription
         {
             public static string name = "VertexDescription";
 
+            public static BlockFieldDescriptor CustomVelocity = new BlockFieldDescriptor(VertexDescription.name, "CustomVelocity", "Add Velocity (OS)", "VERTEXDESCRIPTION_CUSTOMVELOCITY",
```

Comment:
Should we move the (OS) note to the tooltip/documentation? Since it goes against the existing way it is already done, for Normal or Tangent input. Perhaps we would just name the block "Velocity" and be clear in the tooltip that this input is integrated **Additively** in **Object Space**.

Reply:
Sadly there is no tooltip for block. But I suppose you mean on the option to enable this input?

Reply:
ok will do the change. Sadly as I was thinking there is only a tooltip on the option in graph settings, hope users will not be confuse. What I like with normal for example is that you can have the space you want and it is show but the concept will not exist for velocity